docker-compose-prod.yml

```diff
@@ -0,0 +1,51 @@
+version: '2'
+services:
+  nginx:
+    image: jwilder/nginx-proxy
+    ports: 
+      - "80:80"
+      - "443:443"
```

Comment:
not sure, I need to figure that out.  I'll probably do that when we get the domain and I setup letsencrypt